ReactiveTwitterSearch/ViewModel/TwitterSearchViewModel.swift

```diff
@@ -50,7 +50,7 @@ class TwitterSearchViewModel {
         }
 
     timer(Constants.TickIntervalLength, onScheduler: QueueScheduler.mainQueueScheduler).startWithNext { [weak self] _ in
-      self?.tweets.value.forEach { $0.updateTime() }
+        self?.tweets.value.update()
```

Comment:
No worries. I can follow up with an update (ha, no pun intended) back to the original.